docs/source/quickstart_drilldown.rst

```diff
@@ -0,0 +1,135 @@
+:orphan:
+
+.. _quickstart_drilldown:
+
+Quickstart options and troubleshooting
+============================================================
+
+..
+    Eventually, these H2s will probably all be separate articles. For now, I'm 
+    avoiding that so as not to create a bunch of super-skinny pages. 
+
+.. _quickstart_drilldown_install:
+
+Customize and troubleshoot MLflow installation
+---------------------------------------------------
+
+Python library options
+**********************
+
+Rather than the default MLflow library, you can install the following variations:
+
+.. list-table::
+   :widths: 10 20 70
+   :header-rows: 1
+
+   * - Name
+     - ``pip install`` command
+     - Description
+   * - mlflow-skinny
+     - ``pip install mlflow-skinny``
+     - Lightweight MLflow package without SQL storage, server, UI, or data science dependencies.
+   * - mlflow[extras]
+     - ``pip install mlflow[extras]``
+     - MLflow package with all dependencies needed to run various MLflow flavors. These dependencies are listed in doc:`https://github.com/mlflow/mlflow/blob/master/requirements/extra-ml-requirements.txt`
+
+
+
+Python and Mac OS X
+**********************
+
+We strongly recommend using a virtual environment manager on Macs. We always recommend using virtual environments, but they are especially important on Mac OS X because the system ``python`` version varies depending on the installation and whether you've installed the Xcode command line tools.
+
+Virtual environment managers
+****************************
+
+We release MLflow on:
+
+- PyPI (``pip install mlflow``)
+- conda-forge (``conda install -c conda-forge mlflow``)
+- CRAN (``install.packages("mlflow")``)
+- Maven Central (``mlflow-client``, ``mlflow-parent``, ``mlflow-scoring``, ``mlflow-spark``)
+
+R and Java
+**********
+
+See :ref:`installing MLflow for R<R-api>` . For Java, we release to Maven Central. For more information, see :ref:`java_api`.
+
+
+.. _quickstart_drilldown_log_and_load_model:
+
+Save and serve models
+-------------------------
+
+MLflow includes a generic ``MLmodel`` format for saving *models* from a variety of tools in diverse
+*flavors*. For example, many models can be served as Python functions, so an ``MLmodel`` file can
+declare how each model should be interpreted as a Python function in order to let various tools
+serve it. MLflow also includes tools for running such models locally and exporting them to Docker
+containers or commercial serving platforms.
+
+To illustrate this functionality, the ``mlflow.sklearn`` package can log scikit-learn models as
+MLflow artifacts and then load them again for serving. There is an example training application in
+``sklearn_logistic_regression/train.py`` that you can run as follows:
```

Comment:
Can we change this to a link on github master to this example in case someone wants to directly see the code here?

Reply:
DONE